benchmarks/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+manifold="all"
+n_samples=10
+operation="all"
+while getopts m:n:o: flag
+do
+    case "${flag}" in
+        m) manifold=${OPTARG};;
+        n) n_samples=${OPTARG};;
+        o) operation=${OPTARG};;
+    esac
+done
+echo "Manifold: $manifold";
+echo "N Samples: $n_samples";
+echo "Operation: $operation";
+
+
+python generate_benchmark_params.py
+pytest exp/time_exp.py  --benchmark-columns='min, max'  --benchmark-sort='fullname'
+pytest log/time_log.py  --benchmark-columns='min, max'  --benchmark-sort='fullname'
+pytest dist/time_dist.py --benchmark-columns='min, max'  --benchmark-sort='fullname'
```

Comment:
I don't see the dist folder, will this fail?

Reply:
@ninamiolane  sorry. turns out .gitignore had this dist/ folder. So folder got ignored. completely. So let me what it should be done here. Thanks!